torch/_torch_docs.py

```diff
@@ -6725,22 +6725,37 @@ def merge_dicts(*dicts):
 
 add_docstr(torch.ormqr,
            r"""
-ormqr(input, input2, input3, left=True, transpose=False) -> Tensor
+ormqr(input, tau, other, left=True, transpose=False, *, out=None) -> Tensor
 
-Multiplies `mat` (given by :attr:`input3`) by the orthogonal `Q` matrix of the QR factorization
-formed by :func:`torch.geqrf` that is represented by `(a, tau)` (given by (:attr:`input`, :attr:`input2`)).
+Computes the matrix-matrix multiplication of a product of Householder matrices with a general matrix.
 
-This directly calls the underlying LAPACK function `?ormqr`.
-See `LAPACK documentation for ormqr`_ for further details.
+Multiplies a matrix `C` (given by :attr:`other`) with a matrix `Q`,
+where `Q` is represented using Householder reflectors `(input, tau)`.
```

Comment:
If some elements in `tau` are zeros then that means the corresponding Householder vector will be zeroed, so its contribution will not be considered, leaving an identity transformation instead of reflection. If all elements are zero then the implicit Q matrix will be identity. I think this follows from the LAPACK note on the Householder representation and somebody working with this representation should expect such behavior after reading it.

Reply:
This is too niche of a function, so the details of the documentation can be just a link to LAPACK, as you already did. Most probably the correct question would be "why are we exposing a function as specific as this one", but that's a point for some other time.